cmd/start/cmd.go

```diff
@@ -130,6 +130,28 @@ func enqueueMissingBlocks(exportQueue types.HeightQueue, ctx *parser.Context) {
 		panic(fmt.Errorf("failed to get last block from RPCConfig client: %s", err))
 	}
 
+	lastBlockHeightInDB, err := ctx.Database.GetLastBlockHeight()
+	if err != nil {
+		ctx.Logger.Error("failed to get last block height from database", "error", err)
+	}
+
+	// Get the start height, default to the config's height
+	startHeight := cfg.StartHeight
+
+	// Set startHeight to the latest height in database
+	// if is not set inside config.yaml file,
+	if startHeight == 0 {
+		if lastBlockHeightInDB > 0 {
+			startHeight = lastBlockHeightInDB
+		} else {
+			startHeight = 1
+		}
+	}
+
+	if lastBlockHeightInDB > startHeight {
+		startHeight = lastBlockHeightInDB
+	}
```

Comment:
I believe this needs to be deleted, as otherwise `lastBlockHeightInDB` will always override `startHeight` even when that is defined inside the configuration file (which we shouldn't want to, since it makes it useless in all cases)